xhr.bs

```diff
@@ -1884,7 +1886,7 @@ attributes must return the value they were initialized to.
 
 <h3 id=firing-events-using-the-progressevent-interface>Firing events using the {{ProgressEvent}} interface</h3>
 
-<p>To <dfn id=concept-event-fire-progress>fire a progress event</dfn> named <var>e</var> at
+<p>To <dfn id=concept-event-fire-progress>fire a progress event</dfn> named <var>e</var> on
```

Comment:
The convention is "at", as established by fire an event.

Reply:
Got it. I'll use "at" in this patch and fix the existing uses of "on" in a follow-up.